src/cli.rs

```diff
@@ -74,6 +77,12 @@ impl Options {
                 .multiple(true)
                 .conflicts_with("q")
                 .help("Increases the level of verbosity (the max level is -vvv)"))
+            .arg(Arg::with_name("command")
+                .short("e")
+                .multiple(true)
+                .takes_value(true)
+                .allow_hyphen_values(true)
+                .help("Command and args to execute (must be last argument)"))
```

Comment:
Maybe but it would violate the least surprise rule since other terminal emulators (that I tested) have the behavior that I implemented..

```
$ xterm -help
    -e command args ...          command to execute
```
```
$ st --help
usage: st [-aiv] [-c class] [-f font] [-g geometry] [-n name] [-o file]
          [-T title] [-t title] [-w windowid] [[-e] command [args ...]]
```

```
$ urxvt --help
rxvt-unicode (urxvt) v9.21 - released: 2014-12-31
Usage: urxvt [options] [-e command args]
```

Acutally only gnome-terminal (that I tried) uses `-e 'bash --norc'`.